inference-engine/src/transformations/src/transformations/disable_decompression_convert_constant_folding.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/disable_decompression_convert_constant_folding.hpp"
+
+#include "openvino/opsets/opset8.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/rt_info/disable_constant_folding.hpp"
+#include "transformations/rt_info/decompression.hpp"
+#include "itt.hpp"
+
+ov::pass::DisableDecompressionConvertConstantFolding::DisableDecompressionConvertConstantFolding() {
+    MATCHER_SCOPE(DisableDecompressionConvertConstantFolding);
```

Comment:
don't we loose accuracy if we will fold constants in fp16? Why don't we fold the first and then convert to fp16?

Reply:
No, this is not what this transformation does. This transformation disables constant folding of already compressed constants because that is the new format and we don't want them to become fp32 suddenly.